### PR TITLE
Document QGIS_SERVER_IGNORE_BAD_LAYERS

### LINKED ISSUE #5277: Server: add QGIS_SERVER_IGNORE_BAD_LAYERS config option (Request in QGIS)
### Request for documentation
From pull request QGIS/qgis#35738
Author: @elpaso
QGIS version: 3.14

**Server: add QGIS_SERVER_IGNORE_BAD_LAYERS config option**

### PR Description:
Allow to override the default behavior in case of
bad layers (which is to invalidate the whole project).

When set to TRUE, the bad layers are skipped and the
project is considered available.

Followup #8922

Funded by: Geoinformatikbüro Dassau GmbH

### Commits tagged with [need-docs] or [FEATURE]

### PR DESCRIPTION
**Pull request by elpaso**
Fixes #5277

### CODE REVIEW

File: docs/user_manual/working_with_ogc/server/config.rst [hunk 1]
```diff
@@ -176,6 +176,17 @@ numeric values (e.g. in GetFeatureInfo responses). The default value is ``0``.
 * ``1`` or ``true`` (case insensitive)
 
 
+QGIS_SERVER_IGNORE_BAD_LAYERS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+"Bad" layers are layers that cannot be loaded, the default behavior of QGIS Server
```
Comment (havatv) [inline docs/user_manual/working_with_ogc/server/config.rst:182]:
```suggestion
"Bad" layers are layers that cannot be loaded. The default behavior of QGIS Server
```

File: docs/user_manual/working_with_ogc/server/config.rst [hunk 2]
```diff
@@ -176,6 +176,17 @@ numeric values (e.g. in GetFeatureInfo responses). The default value is ``0``.
...
+is to consider a whole project not available if it contains any bad layer.
```
Comment (havatv) [inline docs/user_manual/working_with_ogc/server/config.rst:183]:
```suggestion
is to consider the project as not available if it contains a bad layer.
```

File: docs/user_manual/working_with_ogc/server/config.rst [hunk 3]
```diff
@@ -176,6 +176,17 @@ numeric values (e.g. in GetFeatureInfo responses). The default value is ``0``.
...
+
+The default behavior can be overridden by setting this variable to ``1`` or ``true``,
```
Comment (havatv) [inline docs/user_manual/working_with_ogc/server/config.rst:185]:
```suggestion
The default behavior can be overridden by setting this variable to ``1`` or ``true``.
```

File: docs/user_manual/working_with_ogc/server/config.rst [hunk 4]
```diff
@@ -176,6 +176,17 @@ numeric values (e.g. in GetFeatureInfo responses). The default value is ``0``.
...
+in this case the "bad" layers will be ignored but the project will still be considered
```
Comment (havatv) [inline docs/user_manual/working_with_ogc/server/config.rst:186]:
```suggestion
In this case, "bad" layers will just be ignored, and the project will be considered
```

**Comment by Gustry** [comment]
Thanks for the update. Shouldn't this be backported to 3.10 documentation?

**Comment by elpaso** [comment]
@Gustry yes

**Comment by elpaso** [comment]
@Gustry done (manually)

**Pull request merged**